### DISCUSSION TITLE
sherlock starting issues

### DISCUSSION THREAD

**Discussion by LUCKST13** [sherlock-project/sherlock#1135] [Q&A]
what do i do when kt show#me

 Requirement already satisfied: urllib3<1.27,>=1.21.1 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from requests>=2.22.0->-r requirements.txt (line 7)) (1.26.6)
Using legacy 'setup.py install' for bs4, since package 'wheel' is not installed.
Using legacy 'setup.py install' for lxml, since package 'wheel' is not installed.
Installing collected packages: lxml, colorama, bs4
    Running setup.py install for lxml ... \
what do i do next

**Comment by ErrorBot101** [comment]
Well if you installed all modules then go to Sherlock folder under the main Sherlock folder here you see a file sherlock.py execute by giving them command

$ python3 sherlock.py _username_ or --help 

If you got some module error so, I also want help for modules installation😅😅

**Comment by LUCKST13** [comment]
im trying to install sherlock n when i used this comand
~/sherlock $ pip install -r requirements.txt
Requirement already satisfied: beautifulsoup4>=4.8.0 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 1)) (4.10.0)
Requirement already satisfied: bs4>=0.0.1 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 2)) (0.0.1)
Requirement already satisfied: certifi>=2019.6.16 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 3)) (2021.5.30)
Collecting colorama>=0.4.1
  Using cached colorama-0.4.4-py2.py3-none-any.whl (16 kB)
Collecting lxml>=4.4.0
  Using cached lxml-4.6.3.tar.gz (3.2 MB)
Requirement already satisfied: PySocks>=1.7.0 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 6)) (1.7.1)
Requirement already satisfied: requests>=2.22.0 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 7)) (2.26.0)
Requirement already satisfied: requests-futures>=1.0.0 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 8)) (1.0.0)
Requirement already satisfied: soupsieve>=1.9.2 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 9)) (2.2.1)
Requirement already satisfied: stem>=1.8.0 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 10)) (1.8.0)
Requirement already satisfied: torrequest>=0.1.0 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from -r requirements.txt (line 11)) (0.1.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from requests>=2.22.0->-r requirements.txt (line 7)) (1.26.6)
Requirement already satisfied: idna<4,>=2.5 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from requests>=2.22.0->-r requirements.txt (line 7)) (2.10)
Requirement already satisfied: charset-normalizer~=2.0.0 in /data/data/com.termux/files/usr/lib/python3.9/site-packages (from requests>=2.22.0->-r requirements.txt (line 7)) (2.0.4)
Building wheels for collected packages: lxml
  Building wheel for lxml (setup.py) ... \

kan anyone help me figure this out i being trying for weeks n kant surpass that
